content/zh/blog/2022/istiocon-register/index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "立即注册 IstioCon 2022!"
+description: 会议将于 4 月底举行，前 400 名参与注册者将收到一件会议 T 恤。
+publishdate: 2022-03-21
+attribution: "IstioCon Organizers Committee"
+keywords: [IstioCon,Istio,conference]
+---
+
+以用户为中心的 IstioCon 是业界最受欢迎的服务网格 Istio 的年度的盛会。该活动将于 4 月 25 日至 29 日举行，它将完全采用虚拟的方式在线上进行，现在[免费开放注册](https://events.istio.io/istiocon-2022/)。如果您是前 400 位注册会议的人之一，您就有资格获得会议 T 恤！
```

Comment:
Have you seen the comments in #11956? Do you need another copy here about the comments again?